api/v6/report_server.thrift

```diff
@@ -219,16 +283,16 @@ service codeCheckerDBAccess {
   // file will be also returned
   // PERMISSION: PRODUCT_ACCESS
   SourceFileData getSourceFileData(
-                                   1: i64 fileId,
-                                   2: bool fileContent,
+                                   1: i64      fileId,
+                                   2: bool     fileContent,
                                    3: Encoding encoding)
                                    throws (1: shared.RequestFailed requestError),
 
   // change review status of a bug.
   // PERMISSION: PRODUCT_ACCESS or PRODUCT_STORE
-  bool changeReviewStatus(1: i64 reportId,
-                          2: shared.ReviewStatus status,
-                          3: string message)
+  bool changeReviewStatus(1: i64          reportId,
```

Comment:
Why do you use this indentation style:
```
ParamType     variable,
HugeParamType variable2
```
It's our convention in thrift files for now?
In this case refactor all other function to this form (getSeverityCounts, getReviewStatusCounts etc).

Reply:
I don't know if this is a convention or not. We just started doing this some weeks ago, though it certainly looks better.